jjb/jobs/template.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Jobs definitions.
+---
+- job-template:
+    name: 'kata-containers-{repo}-{os}-{ci_job}'
+    description: |
+      Test pull requests for Kata Containers.
+        Repository - {repo}
+        OS - {os}
+    concurrent: true
+    logrotate:
+      daysToKeep: 30
+      numToKeep: 30
+    node: !include-jinja2: os2node.yaml.inc
+    scm:
+      - git:
+         url: https://github.com/kata-containers/{repo}
+         branches:
+          - '$sha1'
+    triggers:
+      - github-pull-request:
+          auth-id: "katacontainers bot connection"
+          github-hooks: true
+    builders:
+      - shell: |
+          #!/bin/bash
+          set -e
```

Comment:
How about also adding `set -o nounset`? Also, is there a way to set a variable to enable tracing to work with out standard idiom of:

```bash
[ -n "${DEBUG:-}" ] && set -o xtrace
```